lib/tasks/auto_generate_diagram.rake

```diff
@@ -0,0 +1,6 @@
+# NOTE: only doing this in development as some production environments (Heroku)
```

Comment:
Does this mean it runs the generation on application boot locally? If so, I think this should be structured as a rake task that only runs when explicitly called

Reply:
It is configured to run on `db:migrate` in `development` - https://github.com/voormedia/rails-erd/blob/master/lib/tasks/auto_generate_diagram.rake.

Reply:
I think it'd be kinda preferable to only generate when called explicitly rather than rely on being in a specific environment while running a database migration. To Zach's question below, I think it'd make sense to figure out how to do it in CI (whether it's just a check that the diagram is up to date or generating it for viewing).